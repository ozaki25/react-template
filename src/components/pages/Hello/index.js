import React from 'react';
import { Link } from 'react-router-dom';

function Hello() {
  return (
    <main>
      <section>
        <h1>Hello</h1>
        <Link to="/">HOME</Link>
      </section>
    </main>
  );
}

export default Hello;
