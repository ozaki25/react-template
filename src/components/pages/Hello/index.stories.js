import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';
import Hello from './';

const stories = storiesOf('pages', module);
stories.addDecorator(story => <BrowserRouter>{story()}</BrowserRouter>);
stories.add('Hello', withInfo('Hello')(() => <Hello />));
