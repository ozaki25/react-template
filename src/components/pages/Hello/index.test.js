import React from 'react';
import { snapshot } from '../../../../test/helpers';
import Hello from './';

snapshot('Hello', <Hello />);
