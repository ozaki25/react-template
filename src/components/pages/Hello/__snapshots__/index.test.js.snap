// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: Hello 1`] = `
<main>
  <section>
    <h1>
      Hello
    </h1>
    <Link
      replace={false}
      to="/"
    >
      HOME
    </Link>
  </section>
</main>
`;
